webpack.config.js

```diff
@@ -134,9 +134,14 @@ module.exports = {
         chunkFilename: '[id].css'
       }
     ),
+    example('assignments'),
+    example('embedded-signup'),
+    example('header'),
     example('header-itsi'),
```

Comment:
Looks good to me. Just curious as to why the additions for 4 of the moved pages are added here in the webpack config, except for header-itsi, which looks like it was already defined as an "example" plugin. Did it already exist in the examples directory? Was it, perhaps, an earlier specialization of the basic header?

Reply:
Yes header-itsi was already there. I had forgotten to delete it from src/portals/...